onnxruntime/core/platform/windows/telemetry.cc

```diff
@@ -240,6 +260,7 @@ void WindowsTelemetry::LogRuntimePerf(uint32_t session_id, uint32_t total_runs_s
 
   TraceLoggingWrite(telemetry_provider_handle,
                     "RuntimePerf",
+                    TraceLoggingBool(true, "UTCReplace_AppSessionGuid"),
```

Comment:
can you explain what this one is for? why log something that is always true?

Reply:
This is to log appsessionguid, like other telemetry events.

---
In reply to: [393753326](https://github.com/microsoft/onnxruntime/pull/3234#discussion_r393753326) [](ancestors = 393753326)